yang/frr-bfdd.yang

```diff
@@ -219,7 +219,7 @@ module frr-bfdd {
 
     leaf required-echo-receive-interval {
       type uint32 {
-          range "0 | 10000..60000000";
```

Comment:
This is wrong. 0 is a special value that means that the system doesn't support receiving echo packets.

Reply:
changed